validator/client/src/test/java/tech/pegasys/teku/validator/client/duties/BlockProductionDutyTest.java

```diff
@@ -218,7 +227,37 @@ public void forDeneb_shouldCreateAndPublishBlockContents() {
     assertThat(signedBlock.getMessage()).isEqualTo(unsignedBlock);
     assertThat(signedBlock.getSignature()).isEqualTo(blockSignature);
 
-    // TODO Add test for blobs and kzg proofs once added
+    assertThat(signedBlockContents.getKzgProofs()).isPresent();
+
+    final SszList<SszKZGProof> kzgProofsFromSignedBlockContent =
+        signedBlockContents.getKzgProofs().get();
+
+    assertThat(kzgProofsFromSignedBlockContent).isNotEmpty();
+
+    IntStream.range(0, kzgProofsFromSignedBlockContent.size())
+        .forEach(
+            index -> {
+              final SszKZGProof kzgProofFromSignedBlockContents =
+                  kzgProofsFromSignedBlockContent.get(index);
+              final SszKZGProof kzgProofFromUnsignedBlockContents =
+                  kzgProofsFromUnsignedBlockContents.get(index);
+              assertThat(kzgProofFromSignedBlockContents)
+                  .isEqualTo(kzgProofFromUnsignedBlockContents);
+            });
+
+    assertThat(signedBlockContents.getBlobs()).isPresent();
+
+    final SszList<Blob> blobsFromSignedBlockContents = signedBlockContents.getBlobs().get();
+
+    assertThat(blobsFromSignedBlockContents).isNotEmpty();
+
+    IntStream.range(0, blobsFromSignedBlockContents.size())
```

Comment:
Can just compare the two collections (same for proofs) instead of looping through each

Reply:
Done